client/locales/de.json

```diff
@@ -0,0 +1,402 @@
+             "ssid": "SSID",
+             "signal": "Signal",
+             "txBitrate": "TX Bitrate",
+             "settings": "W-LAN Einstellungen",
+             "password": "Passwort",
+             "save": "Neue W-LAN Konfiguration speichern",
+             "connected": "Verbunden",
+             "notConnected": "Nich verbunden",
+             "confirm": "sollen die neuen W-LAN Einstellungen wirklich angewendet werden?",
+             "hint": "Hinweis",
+             "hintText": "Es kann jederzeit zum integrierten WLAN-Hotspot zurückgekehrt werden, indem der Reset-Knopf unter dem Deckel betätigt wird.",
+             "appliedOK": "Neue W-LAN Anmeldeinformationen wurden erfolgreich angewendet. Nach dem Drücken von OK wird die Seite aktualisiert. Sie müssen höchstwahrscheinlich die URL ändern, da der Roboter versucht, eine Verbindung zum neuen W-LAN herzustellen."
+        }
+    },
+    "robot": {
+        "states": {
+            "n1": "Starte",
+            "n2": "Ladegerät nicht angeschlossen",
+            "n3": "Leerlauf",
+            "n4": "Fernbedienung aktiv",
+            "n5": "Reinigung",
+            "n6": "Rückkehr zur Basis",
+            "n7": "Manueller Modus",
+            "n8": "Lade",
+            "n9": "Ladeproblem",
+            "n10": "Pausiert",
+            "n11": "Spotreinigung",
+            "n12": "Fehler",
+            "n13": "Herunterfahren",
+            "n14": "Aktualisierung",
+            "n15": "Ankopplung",
+            "n16": "Zum Ziel gehen",
+            "n17": "Zonenreinigung"
+        },
+        "errors": {
+            "n0": "Kein Fehler",
+            "n1": "Fehler des Laser-Abstandssensors",
+            "n2": "Frontstoßsensor verklemmt",
+            "n3": "Absturzsensoren ausgelöst, tragen Sie den Roboter zu einer anderen Stelle",
+            "n4": "Absturzsensoren verschmutzt",
+            "n5": "Hauptbürste steckt fest",
+            "n6": "Seitenbürste steckt fest",
+            "n7": "Haupträder stecken fest",
+            "n8": "Gerät steckt fest, tragen Sie den Roboter zu einer anderen Stelle",
+            "n9": "Staubsammlbehälter fehlt oder Filter defekt",
+            "n10": "Staubfilter verschmutzt",
+            "n11": "Steckt in Magnetbarriere fest",
+            "n12": "Niedriger Batteriestand",
+            "n13": "Ladefehler",
+            "n14": "Batteriefehler",
+            "n15": "Wandsensor verschmutzt",
+            "n16": "Oberfläche ist nicht flach, tragen Sie den Roboter zu einer anderen Stelle",
+            "n17": "Seitenbürstenfehler, versuchen Sie einen Neustart",
+            "n18": "Sauggebläse defekt, versuchen Sie einen Neustart",
+            "n19": "Ladestation ohne Strom",
+            "n21": "Obere Abdeckung des Laser-Abstandssensors steckt fest",
+            "n22": "Frontstoßsensor verschmutzt",
+            "n23": "Signalabgabebereich an Basis verschmutzt",
+            "n24": "Verbotener Bereich erkannt, setzen Sie den Robot auf eine erlaubte Fläche"
```

Comment:
"n24": "Verbotszone erkannt, stellen Sie den Roboter auf eine erlaubte Fläche"
(typo in the new file - "setellen")